chameleon/src/test/java/org/hibernate/omm/query/OrderByTests.java

```diff
@@ -0,0 +1,71 @@
+package org.hibernate.omm.query;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import org.hibernate.omm.AbstractMongodbIntegrationTests;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * @author Nathan Xu
+ */
+class OrderByTests extends AbstractMongodbIntegrationTests {
+
+    @Test
+    void testOrderBy() {
+        var book1 = new Book();
+        book1.id = 1L;
+        book1.title = "War and Peace";
+        book1.author = "Leo Tolstoy";
+        book1.publishYear = 1869;
+
+        var book2 = new Book();
+        book2.id = 2L;
+        book2.title = "Crime and Punishment";
+        book2.author = "Leo Tolstoy";
+        book2.publishYear = 1869;
```

Comment:
at https://github.com/NathanQingyangXu/jpa-mongodb-mapping/commit/3795c44b3a18cdd3ab57562fd18a5dfd3073c2da, I changed the book name to "The Idiot" and the author to Fyodor Dostoevsk, for I found it was published in the same year as per https://en.wikipedia.org/wiki/1869_in_literature!